tests/languages/lua_test.py

```diff
@@ -0,0 +1,22 @@
+from unittest import mock
+
+from pre_commit.languages import lua
+
+
+def test_existing_bootstrap_environment(tmpdir):
+    """An existing environment is permitted."""
+    lua._bootstrap_environment(str(tmpdir))
+
+
+@mock.patch('pre_commit.languages.lua.cmd_output')
```

Comment:
I prefer to always use `with mock.patch.object(...)` -- `mock.patch('string')` has hard-to-trace side-effects and has been a really annoying source of bugs in the past